app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.author = current_user
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to root_path
```

Comment:
- [ ] Please redirect to the post path instead of the root path

Example: ` redirect_to post_path(@comment.post)`

Reply:
This issue is well resolved! ✔️